positioning2/styles.css

```
@@ -0,0 +1,42 @@
+.parent {
+    width: 40%;
+    height: 30%;
+    background-color: rgb(171, 190, 171);
+    border: 5px dashed rgb(106, 98, 98);
+    position: fixed;
+}
```

Nu stiu daca mai lipsesc niste stiluri la mine dar a trebuii sa setez o diminesiune explicita in pixeli pentru parent
```suggestion
.parent {
    width: 1000px;
    height: 500px;
    background-color: rgb(171, 190, 171);
    border: 5px dashed rgb(106, 98, 98);
    position: fixed;
}
```